cli/src/cli/command/plugin/service.py

```
@@ -0,0 +1,109 @@
+from rich.table import Table
+from rich.tree import Tree
+from typer import Exit
+
+from cli.command.plugin.type import OutputType
+from core.component import Info, PluginRegistry
+
+console = Console()
+
+
+async def list_plugins_async(
+    plugin_type: str | None, output_format: str, detailed: bool
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)